pkgs/development/compilers/llvm/5/default.nix

```diff
@@ -23,7 +23,7 @@ let
       ln -s "${cc}/lib/clang/${release_version}/include" "$rsrc"
       ln -s "${targetLlvmLibraries.compiler-rt.out}/lib" "$rsrc/lib"
       echo "-resource-dir=$rsrc" >> $out/nix-support/cc-cflags
-    '' + stdenv.lib.optionalString stdenv.targetPlatform.isLinux ''
+    '' + stdenv.lib.optionalString (stdenv.targetPlatform.isLinux && tools.clang-unwrapped ? gcc && !(stdenv.targetPlatform.useLLVM or false)) ''
```

Comment:
I still don't really understand the use for `clang.gcc` if clang was built using gcc isn't it equivalent to `stdenv.cc.cc`? It would be nice not to have that conditional attribute IMHO.

Reply:
It is gross, but `clang.gcc` is not necessary the one used to build `clang`. `clang-unwrapped` is target-agnostic, but the `gcc` appears in its `passthru` based on the target.

It's all stupid, but at least it is consistent across LLVM versions now. The right thing to do is always build `libstdc++` and `gcc` separately.

Reply:
Yeah, that would improve multiple things, anyway this was more of a question than a review comment.